transportation-data-publishing/data_tracker/signal_pms_fulcrum.py

```diff
@@ -495,4 +511,20 @@ def main():
 
 if __name__ == "__main__":
 
-    print(main())
+
+    # args = cli_args()
+    # app_name = args.app_name
+    app_name = "data_tracker_prod"
```

Comment:
why the switch to hardcoded string instead of from args?

Reply:
Hi, Mateo, everything under name == main is for testing only. The script launcher will not read it. But I guess I should comment everything out under name==main to increase some readability.